akka-http-core/src/test/scala/akka/http/impl/engine/client/ClientConfigurationSpec.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (C) 2009-2014 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka.http.impl.engine.client
+
+import akka.stream.testkit.AkkaSpec
+
+class ClientConfigurationSpec extends AkkaSpec {
```

Comment:
+1 for adding those if the above configuration duplication is really the only way to go.